Doc/howto/regex.rst

```diff
@@ -844,7 +844,7 @@ backreferences in a RE.
 
 For example, the following RE detects doubled words in a string. ::
 
-   >>> p = re.compile(r'(\b\w+)\s+\1\b')
+   >>> p = re.compile(r'\b(\w+)\s+\1\b')
```

Comment:
And in other occurrences please.

Reply:
Wouldn't moving \b in the other occurrences change the regular expression?

Reply:
I meant examples with named groups below.

Reply:
I meant the same as well.

Reply:
This will change the regular expression, but the changed regular expression will be equivalent the original one.